### DISCUSSION TITLE
I am so frustrated with tables

### DISCUSSION THREAD

**Discussion by SebastianStehle** [GitbookIO/community#658] [0. General bugs and questions]
In general I have the impression you change things all the time and very often things break. The quality of the editor is 

One thing that is really frustrating are tables. I have no clue how they column size actually works. There is no visual feedback if a table column is using percentage based sizing or absolute sizing and the auth formatting does not work.

For example I have this table in the editor:

![image](https://github.com/user-attachments/assets/6df51023-fb48-4e28-b1b4-dc5a3f1a3d79)

and this is the actual output:

![image](https://github.com/user-attachments/assets/54a0781d-0787-40ed-83a1-0cd6bf1803b1)

I have no clue how to fix this table. I have used "Reset column sizing" ... didn't help. When I change the column size manually the table looses its fixed width that was (afaik) the default behavior for many years.

**Comment by aikoii** [comment]
Hey @SebastianStehle 👋! The issue with the table column sizing differences between the published and in-app versions has been fixed.
